pkg/kube/client.go

```diff
@@ -1096,6 +1101,10 @@ func (c *client) ServicesForSelector(ctx context.Context, namespace string, labe
 	})
 }
 
+func (c *client) SetDefaultApplyNamespace(namespace string) {
+	c.defaultApplyNamespace = namespace
```

Comment:
Should we make it `default` like kubectl

Reply:
Users can set it themselves. I don't think its appropriate to have an implicit default since it can lead to accidentally deploying to `default` unintentionally